src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,12 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder str = new StringBuilder("");
```

Comment:
str - is not very informative naming, lets change it

Reply:
done